docs/spec/staking/TODO.md

```diff
@@ -0,0 +1,5 @@
+
+ - state.md needs updates to include 
```

Comment:
Maybe we track this in issues, not in the repo itself. Also I don't know if we should ship this spec with these crucial pieces missing. cc @alexanderbez @cwgoes

Reply:
I don't think these pieces are crucial, they're pretty straightforward implementation wise - but they will make the staking implementation spec complete, I don't think they should be included in this PR but supplementary PRs - I do agree that they should be github issues probably however in addition to this (specifically so auditors can see what may be incomplete)

Reply:
tracking issue: https://github.com/cosmos/cosmos-sdk/issues/3324